index.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>99 Bottles</title>
+</head>
+<body>
+  <h1>99 Bottlesof Beer</h1>
+  <ul> class="bottles"
+    <li> <%= @Bottles %> </li>
+    <li>Take One Down<a href="/:num_bottles"></a>
```

Comment:
This variable you're using on line 10 is different thatn the one that you instantiated in `app.rb`